server/lib/accounts.coffee

```diff
@@ -136,6 +127,35 @@ Accounts.validateLoginAttempt (login) ->
 	return true
 
 Accounts.validateNewUser (user) ->
+	# bypass for livechat users
+	if user.type is 'visitor'
+		return true
+
 	if RocketChat.settings.get('Accounts_Registration_AuthenticationServices_Enabled') is false and RocketChat.settings.get('LDAP_Enable') is false and not user.services?.password?
 		throw new Meteor.Error 'registration-disabled-authentication-services', 'User registration is disabled for authentication services'
 	return true
+
+
+Accounts.validateNewUser (user) ->
```

Comment:
Why using the `Accounts.validateNewUser` again, instead of using it from line 129?